deploy/linea/deploy_linea_lseth.ts

```
@@ -0,0 +1,44 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { isDeployed, logStep, logStepEnd } from "../../ts-utils/helpers/index";
+import { verify } from "../../scripts/helpers";
```

ESLint

[error] "../../scripts/helpers" is not published.

(node/no-unpublished-import)